apps/dashboard/src/components/auth/customize-inbox-playground.tsx

```diff
@@ -149,12 +154,62 @@ function ColorPickerSection({ form }: { form: UseFormReturn<InboxPlaygroundFormD
   );
 }
 
+const extensions = [EditorView.lineWrapping];
+const basicSetup = {
+  defaultKeymap: true,
+};
+
 function NotificationConfigSection() {
+  const { control } = useFormContext();
+
   return (
     <div className="flex flex-col gap-1 p-1">
-      <div className="flex gap-1">{getComponentByType({ component: UiComponentEnum.IN_APP_SUBJECT })}</div>
-      {getComponentByType({ component: UiComponentEnum.IN_APP_BODY })}
-      {getComponentByType({ component: UiComponentEnum.IN_APP_BUTTON_DROPDOWN })}
```

Comment:
The `getComponentByType` is the in-app editor specific util that we shouldn't use in any other context. 
So I refactored the code to use the form fields instead.